src/styles/main.css

```diff
@@ -1,10 +1,22 @@
 html body {
```

Comment:
you probably meant for this to be
```suggestion
html,
body {
```